### DISCUSSION TITLE
Character Creation using MULTISESSION_MODE: 2

### DISCUSSION THREAD

**Discussion by Supersquirtle6** [evennia/evennia#2518] [Evennia Questions]
Hi, I'm using multisession mode 2 and I need help adding more things to the character creation of Multisession mode 2.

If someone could point me in the right direction that would be great!

Thanks!

**Comment by Griatch** [comment]
The `MULTISESSION_MODE=2` 'chargen' is very simple and containsonly two elements: 
- the `OOCLook` command ([evennia.commands.default.account.CmdOOCLook](https://github.com/evennia/evennia/blob/master/evennia/commands/default/account.py#L88)) which shows the little menu when you connect in that multisession mode
- The `charcreate`command ([evennia.commands.default.account.CmdCharCreate](https://github.com/evennia/evennia/blob/master/evennia/commands/default/account.py#L123)) which allows you to create and describe a new character. 

Either of these can be replaced like any other commands; you just need to add your custom variations to the `AccountCmdset` rather than on the `CharacterCmdset`. The general principle of overriding commands is found [here](https://www.evennia.com/docs/latest/Adding-Command-Tutorial.html?highlight=new%20command) (too replace a default command, just give your custom version the same `key` as the default). 

Hope that helps!
